docs/distributions/arch/installation.md

```diff
@@ -7,7 +7,7 @@ You will need:
 
 - USB drive with at least 1GB
 - A way to plug it into your Mac (USB-C isn't USB-A)
-- A wired internet connection (i.e. USB-C to Enternet dongle) or wifi. If you need to install via wifi, you may use [this iso](https://github.com/Redecorating/archiso-mbp/releases), which has everything needed to follow the [wifi guide](https://wiki.t2linux.org/guides/wifi) and use wifi in the live environment.
+- A wired internet connection (i.e. USB-C to Enternet dongle) or wifi. If you need to install via wifi, you may use [this iso](https://mirror.funami.tech/archiso-mact2/), which has everything needed to follow the [wifi guide](https://wiki.t2linux.org/guides/wifi) and use wifi in the live environment.
```

Comment:
Could you change this link to your repo's releases page? https://github.com/NoaHimesaka1873/archiso-t2/releases

Reply:
Just did that!